readme/desktop.md

```diff
@@ -1,5 +1,103 @@
 # Desktop application
 
+
+
+For general information relevant to all the applications, see also [Joplin home page](https://joplinapp.org).
+
+# An illustrated Guide to Using Joplin.  
+•	Why note taking at all?  
+o	It makes it easier to find small bits of information that don’t warrant a full document: the VIN number and tire air pressure for your car; Veterinary data for pets, the phone number and ordering website for a good pizza place, all the way up to major projects of any type.
+•	Why Joplin over other note managers?  
+
+o	OneNote – Too many limitations, and too many of the wrong capabilities. Learning curve is high.
+
+o	EverNote – Too buggy.
+
+o	Google Keep. Too slow, no offline/desktop app. No way to nest notes. 
+
+o	Also, these above leave you beholden to a corporate data format which may or may not be in your best interest.
+o	Joplin features are straightforward: the features it has, are pretty bulletproof.  If a feature is included, it works and consistently.
```

Comment:
What's up with the bullet points `o` and `•`? They are not markdown.

Reply:
Bullet points are a vestige of the original edit.  They showed up when I pasted..  Is there a way to edit the doc as it stands now so I can fix this and Daerexa's comments?